packages/ui5-middleware-cfdestination/lib/cfdestination.js

```diff
@@ -19,62 +19,62 @@ const log = require("@ui5/logger").getLogger("server:custommiddleware:cfdestinat
  * @returns {function} Middleware function to use
  */
 module.exports = function ({ resources, options }) {
-    request.debug = options.configuration.debug; // pass debug flag on to underlying request lib
-    process.env.XS_APP_LOG_LEVEL = options.configuration.debug ? 'DEBUG' : 'ERROR';
+    request.debug = options.configuration.debug // pass debug flag on to underlying request lib
+    process.env.XS_APP_LOG_LEVEL = options.configuration.debug ? 'DEBUG' : 'ERROR'
     // read in the cf config file
-    const xsappConfig = JSON.parse(fs.readFileSync(options.configuration.xsappJson, 'utf8'));
-    const xsappPath = options.configuration.xsappJson.replace("xs-app.json", "");
+    const xsappConfig = JSON.parse(fs.readFileSync(options.configuration.xsappJson, 'utf8'))
+    const xsappPath = options.configuration.xsappJson.replace("xs-app.json", "")
 
     // the default auth mechanism is set to none but the user can pass an auth method using the options
-    xsappConfig.authenticationMethod = options.configuration.authenticationMethod || "none";
+    xsappConfig.authenticationMethod = options.configuration.authenticationMethod || "none"
 
-    let regExes = [];
+    let regExes = []
     xsappConfig.routes = xsappConfig.routes
-        .filter((route) => !route.localDir && (options.configuration.allowServices || !route.service)); //ignore routes that point to web apps as they are already hosted by the ui5 tooling
+        .filter((route) => (options.configuration.allowLocalDir || !route.localDir) && (options.configuration.allowServices || !route.service)) //ignore routes that point to web apps as they are already hosted by the ui5 tooling
```

Comment:
there should be also an options.configuration check in general as in case of missing configuration an error occurs - even if this is not the case somewhere - some people contacted me already therefore - can we use the middleware with default values only without configuration? in this case it would make sense to have a options.configuration check.